pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -281,11 +288,19 @@ func (ds *dockerService) getIP(sandbox *dockertypes.ContainerJSON) (string, erro
 		// reporting the IP.
 		return "", nil
 	}
-	if IP, err := ds.getIPFromPlugin(sandbox); err != nil {
-		glog.Warningf("%v", err)
-	} else if IP != "" {
-		return IP, nil
+
+	// Don't bother getting IP if networking isn't ready
```

Comment:
Sounds good.

For the long run, we should probably also checkpoint the actual name we've built for the container (which should always be unique), so that we can always look it up.
/cc @freehan